packages/frontend/core/src/components/pure/cmdk/main.tsx

```diff
@@ -171,7 +172,7 @@ export const CMDKContainer = ({
   const isInEditor = pageMeta !== undefined;
   const [opening, setOpening] = useState(open);
   const { syncing, progress } = useDocEngineStatus();
-  const [showLoading, setShowLoading] = useState(false);
+  const [showLoading, setShowLoading] = useDebouncedState(false, 500);
```

Comment:
oops, maybe you need to use `useDebouncedValue` in this case

Reply:
Do you mean this?
```
 const [showLoading, setShowLoading] = useDebouncedState(false, 500);
  const deferredShowLoading = useDeferredValue(showLoading);
    useEffect(() => {
    setShowLoading(syncing);
  }, [syncing, setShowLoading]);
  
  return (
   {deferredShowLoading ? (
   ...
```